docs/computers/index_computers.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Computers Overview
+sidebar_label: Overview
+sidebar_position: 1
+---
+
+<center>
+  <figure>
+    <img
+      src="/img/about_images/about_4.jpg"
+      width="1000"
+    />
+    <figcaption>Jackal with a water cooled Intel i9 processor and Nvidia 1050</figcaption>
+  </figure>
+</center>
+
+<br />
+
+Clearpath's robots use ROS or ROS2, which require the robot to have a computer running an Operating System like Ubuntu.
+We mostly use mini-ITX computers with Intel Core i3 and i7 processors, but we can run ROS on the major architectures including:
+
+- Intel Core i series
+- AMD Ryzen series
```

Comment:
Have we tried this anytime recently? Mostly just curious. Agreed that it should work.

Reply:
I'm not aware of any AMD builds in the past year. This was a more common request ~3 years ago when the Ryzen line was notably faster than Intel.

Should AMD be removed from this page?

Reply:
Maybe let's remove AMD for now